### PR TITLE
adding category hero component

### PR DESCRIPTION
**Pull request by chenrygit**

### CODE REVIEW

**Comment by chenrygit** [comment]
This new component is for the category page hero/heading area used in user stories CACR-161 (agency) and CACR-162 (corporate).  The same new component applies to both user stories.

Please note:  I wasn't able to specifically use "Open Sans Light", so I got as close as I could using the currently inherited Google web font.

File: new-components/category-page-hero.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+</style>
+
+<style>
+  .v2 .category-hero {
+    position:relative;
+    background-size:cover;
+    background-color: #f1f1f1;
+    width:100%;
+    height:210px;
```
Comment (juliemtaylor) [inline new-components/category-page-hero.html]: Spec indicates this image should be 275px high.

File: new-components/category-page-hero.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  .v2 .category-hero-container {
+    display:block;
+    padding:20px;
+    margin:0 auto;
+    width:960px;
+    height:210px;
+  }
+  .v2 .category-hero-heading {
+    position:absolute;
+    bottom:-50px;
```
Comment (juliemtaylor) [inline new-components/category-page-hero.html]: Can we push up the headline so the center of the words are aligned with the bottom of the image?

File: new-components/category-page-hero.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    margin-left:40px;
+    background:#23669d;
+    padding:12px 40px;
```
Comment (juliemtaylor) [inline new-components/category-page-hero.html]: The height of the headline is closer to 95px. Spec is 90px.

File: new-components/category-page-hero.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    border:5px solid #fff;
+    font-size:45px;
+    font-weight:lighter;
```
Comment (juliemtaylor) [inline new-components/category-page-hero.html]: Should this be `font-weight: 300;`?

**Pull request merged**